src/System.Private.CoreLib/src/System/Threading/EventWaitHandle.cs

```diff
@@ -84,7 +84,7 @@ public bool Reset()
         {
             // The field value is modifiable via <see cref="SafeWaitHandle"/>, save it locally to ensure that ref modification
             // is done on the same instance
-            SafeWaitHandle waitHandle = _waitHandle;
```

Comment:
`_waitHandle` is more in line with the coding convention. Should we change the CoreCLR side instead?